README.md

```diff
@@ -170,6 +170,7 @@ Properties:
 - `tag`: Optional. Tag name.
 - `develop`: Optional. Boolean, can be toggled on/off to activate/deactivate a package. If activated, then deactivated afterwards, the package gets removed from `jsconfig` maintaining the synchronization with `mrs.developer.json`. Default is `true`.
 - `output`: Optional. Output directory override per repository.
+- `noDepth`: Optional. Used together with `tag` or `branch`, it creates a partial clone defaulting to the tag or branch specified.
```

Comment:
noDepth is a bad name now that we changed it to use `--filter=blob:none` instead of `--depth=1`. How about `filterBlobs`

We could probably turn it on by default, but it makes sense to test it more in a release first.

Reply:
@davisagli ok, let's make it opt-in for now, let's have feedback before moving it to default.